routes/v0/contacts.js

```diff
@@ -1,27 +1,27 @@
-const { Router } = require('express');
 const models = require('../../models');
+const { SafeRouter, openEndpoint } = require('../../permissions');
 
 const { Contact, Case, sequelize } = models;
 const ContactController = require('../../controllers/contact-controller')(Contact);
 const CaseController = require('../../controllers/case-controller')(Case, sequelize);
 
-const contactsRouter = Router();
+const contactsRouter = SafeRouter();
 
-contactsRouter.get('/', async (req, res) => {
+contactsRouter.get('/', openEndpoint, async (req, res) => {
   const { accountSid } = req;
   const contacts = await ContactController.getContacts(req.query, accountSid);
   res.json(contacts);
 });
 
 // example: curl -XPOST -H'Content-Type: application/json' localhost:3000/contacts -d'{"hi": 2}'
-contactsRouter.post('/', async (req, res) => {
+contactsRouter.post('/', openEndpoint, async (req, res) => {
   const { accountSid } = req;
 
   const contact = await ContactController.createContact(req.body, accountSid);
   res.json(contact);
 });
 
-contactsRouter.put('/:contactId/connectToCase', async (req, res) => {
+contactsRouter.put('/:contactId/connectToCase', openEndpoint, async (req, res) => {
   const { accountSid } = req;
   const { contactId } = req.params;
   const { caseId } = req.body;
```

Comment:
If I've understood this correctly, all the endpoints that use the `openEndpoint` middleware are going to be public, which means that everyone will be able to access these endpoints.
If that is the case I would suggest changing the name of `openEndpoint` to something like: `publicEndpoint` or `authorizedEnpoint` which makes more sense. What do you think?